### ISSUE TITLE
Change hidden folders to rely on in-app settings instead of .nomedia

### ISSUE THREAD

**Issue by Naomitr0n** [FossifyOrg/Gallery#131]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Gallery/issues) or [closed](https://github.com/FossifyOrg/Gallery/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Gallery/discussions) or [closed](https://github.com/FossifyOrg/Gallery/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

When you select a folder and try to hide it, a message appears saying that it needs to add a .nomedia file.

Ideally it wouldn't need to add this file, and instead changing visibility within Gallery would not affect any other app. Adding a toggle to view hidden folders in the three-dot menu that is disabled by default every time the app opens would still allow viewing of these folders.

### Why do you want this feature?

There are a very large number of reasons this feature is useful.

As one example, I have a TON of one-file folders of album art that Gallery is showing that I'd like to hide. Adding a .nomedia file would hide ALL my music from music players, which is (obviously) quite bad.

As another example, I don't want to see every folder with videos in them (some, yes, but certainly not *all*) but can't hide them without many video editors not being able to see them.

Yet another, I also have a few folders from apps like Snapseed, SquareDroid, etc. that I can't hide but don't need nor want to see the contents of. Sometimes apps will create their own folder and put their own junk in that folder.

Then there's just pictures and videos I go to a file explorer to see instead of the Gallery app for whatever reason. Whether it's sensitive info (scans of sensitive documents, ID cards, etc.), pics not exactly safe for kids (folders of medical procedures), pics I don't want to get rid of, but also doesn't want to see all the time (loved ones who have passed).

And that's before you get to the use case of people who use multiple galleries to see different sets of folders. Maybe an edge case, but I know I certainly let the more nuanced folders be visible in a gallery app I don't normally use so the pics I want to prioritize don't get hidden in the muck.

I really only want to see pictures I've taken, downloaded, or pictures people have sent me, without all the random other garbage, and without having to worry about me hiding it from Gallery causing other apps to not see any of what I've hidden.

**Comment by Aga-C** [comment]
Check the 7th entry in the FAQ. Isn't it what you're requesting?

**Comment by Naomitr0n** [comment]
I don't know how I missed that checkbox when submitting the issue, I thought I got them all carefully, I do apologize for that. I'm not used to FAQ's being under the About menu.

Looking at the menu again, I absolutely see why I missed it:
![Screenshot_20240216-125356_1](https://github.com/FossifyOrg/Gallery/assets/81192719/a49e9fa3-2dbb-43fa-aff4-c8f12aefc938)


Ordering and labeling isn't the most consistent. Enhancement request: change "Exclude" to "Exclude Folder", and move the "Hide Folder " option to be directly above the Exclude Folder option so they're grouped together? I didn't try the Exclude option (which worked) but also didn't think it had both features with the labeling difference.

Something like:
> Reorder
> Rename
> Copy To
> Move To
> Select All
> Create Shortcut
> Hide Folder
> Exclude Folder
> Lock Folder
> Change Cover Image

In the list above, I also moved "Select All" higher to be consistent with most cut/copy/paste/select-style  grouping options, tho I'm not sure the best place to put it. I do think a menu reorder/relabel in general could help reduce confusion however.
